README.md

```diff
@@ -6,6 +6,8 @@ This package contains the Golang interface for using the Windows [Host Compute S
 
 It is primarily used in the [Moby Project](https://github.com/moby/moby), but it can be freely used by other projects as well.
 
+This is a code-only repository and does not have a stable release. 0.8.6 is the last tagged release, and it is out of date for ContainerD and Kubernetes. Downstream projects should vendor in this codebase using commit ids instead of tags, then validate fixes and do a full test pass of the whole project.
```

Comment:
I would prefer we say something along the lines of: "The tags are associated with a vendoring scheme that is only intended for usage by Docker and the v1 `container` and `process` API's. This vendoing scheme is not suitable for ContainerD or Kubernetes which should use commit id's ...". 

Feel free to re-word this and prettify it but that is what those numbers are for. Its not for anything other than that and its all we test them for.

Reply:
Updated - can you review again @jterry75

Reply:
@jterry75 i see a new release `0.8.7` that says `This is the v0.8.7 release of containerd-shim-runhcs-v1.exe for containerd` So is the statement above still true?

Reply:
v0.8.7 is ok for the shim because that is a binary dependency based on a stable API contract (Runtime V2) from containerd. The intent is to make `vendor` based commits (other than Docker) use commit id's

Reply:
@PatrickLang - Can you update this to be a little more generic and reflect the confusion. The goal of our releases is to:
1. Provide a vendor based commit tag for Docker which is code based.
2. Provide a binary release version tag for the stable release of the containerd shim supported in containerd releases.
3. All others are expected to use commit hashes because we are not verifying that when we make changes to a code dependency that we did not break them. We are only verifying this with Docker based vendoring.

Reply:
@jterry75 - I'm not sure I understand where Kubernetes falls into 1-3 based on the following:

- Docker-EE basic is still in use
- Kubernetes makes extra calls not supported in Docker through HCS for gathering metrics, network stats, and HNS configuration. Those rely on hcsshim

Reply:
I guess that would put Kubernetes in the 3rd case, since they're carrying a code dependency on hcsshim directly. It's not affected by Docker's use of hcsshim, and doesn't (for example) imply they must use the *same* hcsshim version that Docker users.

Reply:
Correct